### DISCUSSION TITLE
SMB-C authentication error: client not found

### DISCUSSION THREAD

**Discussion by msteffenDev** [Digitaler-Impfnachweis/certification-apis#50] [Q&A] [answered]
We get the error "Client not found".  What could be the reason?

Thanks.

Her is the command with the results:

curl --request GET 'https://id.ru.impfnachweis.info/auth/realms/ubirch-certify/protocol/openid-connect/auth?redirect_uri=connector://authenticated&response_type=code&scope=openid&client_id=cert-auth&nonce=123456789' -v 
Note: Unnecessary use of -X or --request, GET is already inferred.
*   Trying 10.30.17.10...
* TCP_NODELAY set
* Connected to id.ru.impfnachweis.info (10.30.17.10) port 443 (#0)
* ALPN, offering h2
* ALPN, offering http/1.1
* Cipher selection: ALL:!EXPORT:!EXPORT40:!EXPORT56:!aNULL:!LOW:!RC4:@STRENGTH
* successfully set certificate verify locations:
*   CAfile: /etc/ssl/cert.pem
  CApath: none
* TLSv1.2 (OUT), TLS handshake, Client hello (1):
* TLSv1.2 (IN), TLS handshake, Server hello (2):
* TLSv1.2 (IN), TLS handshake, Certificate (11):
* TLSv1.2 (IN), TLS handshake, Server key exchange (12):
* TLSv1.2 (IN), TLS handshake, Server finished (14):
* TLSv1.2 (OUT), TLS handshake, Client key exchange (16):
* TLSv1.2 (OUT), TLS change cipher, Client hello (1):
* TLSv1.2 (OUT), TLS handshake, Finished (20):
* TLSv1.2 (IN), TLS change cipher, Client hello (1):
* TLSv1.2 (IN), TLS handshake, Finished (20):
* SSL connection using TLSv1.2 / ECDHE-RSA-AES256-GCM-SHA384
* ALPN, server accepted to use h2
* Server certificate:
*  subject: 1.3.6.1.4.1.311.60.2.1.3=DE; 1.3.6.1.4.1.311.60.2.1.2=Nordrhein-Westfalen; 1.3.6.1.4.1.311.60.2.1.1=Koeln; businessCategory=Private Organization; serialNumber=HRB 82994; C=DE; ST=Nordrhein-Westfalen; L=Koeln; postalCode=50670; street=Im Mediapark 5; O=ubirch GmbH; CN=api.ru.impfnachweis.info
*  start date: May 10 14:15:13 2021 GMT
*  expire date: May 13 14:15:13 2022 GMT
*  subjectAltName: host "id.ru.impfnachweis.info" matched cert's "id.ru.impfnachweis.info"
*  issuer: C=DE; O=D-Trust GmbH; CN=D-TRUST SSL Class 3 CA 1 EV 2009
*  SSL certificate verify ok.
* Using HTTP2, server supports multi-use
* Connection state changed (HTTP/2 confirmed)
* Copying HTTP/2 data in stream buffer to connection buffer after upgrade: len=0
* Using Stream ID: 1 (easy handle 0x7f8e6c808a00)
> GET /auth/realms/ubirch-certify/protocol/openid-connect/auth?redirect_uri=connector://authenticated&response_type=code&scope=openid&client_id=cert-auth&nonce=123456789 HTTP/2
> Host: id.ru.impfnachweis.info
> User-Agent: curl/7.54.0
> Accept: */*
> 
* Connection state changed (MAX_CONCURRENT_STREAMS updated)!
< HTTP/2 400 
< content-language: en
< content-security-policy: frame-src 'self'; frame-ancestors 'self'; object-src 'none';
< content-type: text/html;charset=utf-8
< date: Tue, 01 Jun 2021 14:19:27 GMT
< referrer-policy: no-referrer
< server: istio-envoy
< strict-transport-security: max-age=15552000; includeSubDomains; preload
< x-content-type-options: nosniff
< x-envoy-decorator-operation: ruproxy.ubirch-prod.svc.cluster.local:8080/*
< x-envoy-upstream-service-time: 22
< x-frame-options: SAMEORIGIN
< x-robots-tag: none
< x-xss-protection: 1; mode=block
< content-length: 2102
< 
    <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
            "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
    <html xmlns="http://www.w3.org/1999/xhtml" class="login-pf">

    <head>
        <meta charset="utf-8">
        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
        <meta name="robots" content="noindex, nofollow">

                <meta name="viewport" content="width=device-width,initial-scale=1"/>
        <title>Log in to Ubirch Certify App Login</title>
        <link rel="icon" href="/auth/resources/mrpuj/login/Ubirch-simple/img/favicon.ico"/>
                <link href="/auth/resources/mrpuj/common/keycloak/node_modules/patternfly/dist/css/patternfly.min.css" rel="stylesheet"/>
                <link href="/auth/resources/mrpuj/common/keycloak/node_modules/patternfly/dist/css/patternfly-additions.min.css" rel="stylesheet"/>
                <link href="/auth/resources/mrpuj/common/keycloak/lib/zocial/zocial.css" rel="stylesheet"/>
                <link href="/auth/resources/mrpuj/login/Ubirch-simple/css/login-28042021.css" rel="stylesheet"/>
    </head>

    <body class="">

    <div class="header">
        <a class="logo" href="https://ubirch.de">
            <img src="/auth/resources/mrpuj/login/Ubirch-simple/img/Ubirch_Logo_weiss.svg" alt="logo"></a>
        <div id="kc-header-wrapper"
             class="">Ubirch Certify App Login</div>
    </div>

    <div class="login-pf-page">
        <div id="kc-header" class="login-pf-page-header">
            <div id="kc-header-wrapper"
                 class="">Ubirch Certify App Login</div>
        </div>
        <div class="card-pf ">
            <header class="login-pf-header">
                        <h1 id="kc-page-title">        We are sorry...
</h1>
            </header>
            <div id="kc-content">
                <div id="kc-content-wrapper">

        <div id="kc-error-message">
            <p class="instruction">Client not found.</p>
        </div>

                </div>
            </div>

        </div>
    </div>
    </body>
    </html>
* Connection #0 to host id.ru.impfnachweis.info left intact

**Comment by thinkberg** [comment, marked as answer]
There has been a change in the client_id: https://github.com/Digitaler-Impfnachweis/certification-apis/commit/64b236ea2df4d368a45eb6232cb31243f80e69f5

Please change it to user-access-ti and it should work again.

**Reply by msteffenDev** [reply]
it works, thank you
